WorkloadManagementSystem/Executor/JobScheduling.py

```diff
@@ -255,14 +255,14 @@ def __getSitesRequired( self, jobState ):
       self.jobLog.info( "Banned %s sites" % ", ".join( bannedSites ) )
 
     sites = manifest.getOption( "Site", [] )
-    #TODO: Only accept known sites after removing crap like ANY set in the original manifest
+    # TODO: Only accept known sites after removing crap like ANY set in the original manifest
     sites = [ site for site in sites if site.strip().lower() not in ( "any", "" ) ]
 
     if len( sites ) == 1:
       self.jobLog.info( "Single chosen site %s specified" % ( sites[0] ) )
 
     if sites:
-      self.jobLog.info( "Multiple sites requested: %s" ','.join( sites ) )
+      self.jobLog.info( "Multiple sites requested: %s" % ','.join( sites ) )
```

Comment:
This is the only change!
Strangely this typo was equivalent to 
"Multiple sites requested: %s,".join( sites )
because

"this is " "a string" == "this is a string"

dangerous!